js/index.js

```diff
@@ -1,20 +1,107 @@
 // Mobile menu
 
 const navbarUl = document.querySelector('nav.navbar > ul');
+const navbarTitle = document.querySelector('nav.navbar > a#welcome');
 const navbarMenuBtn = document.querySelector('nav.navbar > a#menu-btn');
 const navbarMenuBtnClose = document.querySelectorAll(
   'nav.navbar > ul > li > a',
 );
 
+function getStyle(element, name) {
+  const ComputedStyle = window.getComputedStyle
+    ? window.getComputedStyle(element, null).getPropertyValue(name) : null;
+  return element.currentStyle ? element.currentStyle[name] : ComputedStyle;
+}
+
 function showNavUl() {
+  if (getStyle(navbarTitle, 'display') === 'none') return;
+
   if (navbarUl.classList.contains('show')) {
     navbarUl.classList.remove('show');
   } else {
     navbarUl.classList.add('show');
   }
 }
+// popup modal
+
+function buildWork(
+  title,
+  description,
+  featured,
+  technologies,
+  linkLive,
+  linkSource,
+) {
+  return {
+    title,
+    description,
+    featured,
+    technologies,
+    linkLive,
+    linkSource,
+  };
+}
+function BuildGenericWork(title) {
+  return buildWork(
+    title,
+    `${title} Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia dolor error nam deleniti tempora excepturi iure animi aut, repellat amet aliquam illo quos eaque? Officiis ullam accusamus est soluta obcaecati?`,
+    'assets/works/Snapshoot-Portfolio.png',
+    [
+      'Ruby on rails',
+      'css',
+      'JavaScript',
+      'html',
+    ],
+    'https://carlosmuerto.github.io/Portfolio/',
+    'https://github.com/carlosmuerto/Portfolio',
+  );
+}
+const works = [
+  BuildGenericWork('Multi-Post Stories Gain+Glory 01'),
+  BuildGenericWork('Multi-Post Stories Gain+Glory 02'),
+  BuildGenericWork('Multi-Post Stories Gain+Glory 03'),
+];
```

Comment:
- [ ] You have followed this requirement somehow but it would be better if you would make your array of objects something like this [file](https://github.com/jaflih/Portfolio/blob/main/projects.js) and render your cards dynamically :blush: